saleor/plugins/webhook/tests/subscription_webhooks/payloads.py

```diff
@@ -232,14 +232,26 @@ def generate_permission_group_payload(group):
     }
 
 
-def generate_invoice_payload(invoice):
-    return {
+def generate_invoice_payload(invoice, order=None):
+    order_id = None
+    if invoice.order:
+        order_id = graphene.Node.to_global_id("Order", invoice.order.id)
```

Comment:
unnecessary db call, we should go with
```suggestion
    if invoice.order_id:
        order_id = graphene.Node.to_global_id("Order", invoice.order_id)
```

Reply:
Great catch, thanks!